documentation/modules/cruise-control/con-optimization-goals.adoc

```diff
@@ -71,16 +71,21 @@ An optimization proposal that does _not_ satisfy all the hard goals is rejected
 NOTE: For example, you might have a soft goal to distribute a topic's replicas evenly across the cluster (the topic replica distribution goal).
 Cruise Control will ignore this goal if doing so enables all the configured hard goals to be met.
 
-In Cruise Control, the following xref:main-goals[main optimization goals] are preset as hard goals:
+In Cruise Control, the following xref:main-goals[main optimization goals] are hard goals:
 
 [source]
 RackAwareGoal; ReplicaCapacityGoal; DiskCapacityGoal; NetworkInboundCapacityGoal; NetworkOutboundCapacityGoal; CpuCapacityGoal
 
-You configure hard goals in the Cruise Control deployment configuration, by editing the `hard.goals` property in `Kafka.spec.cruiseControl.config`.
+You configure which hard goals will be enforced to execute in the Cruise Control deployment configuration, by editing the `hard.goals` property in `Kafka.spec.cruiseControl.config`.
 
-* To inherit the preset hard goals from Cruise Control, do not specify the `hard.goals` property in `Kafka.spec.cruiseControl.config`
+* To enforce that all the hard goals are executed, do not specify the `hard.goals` property in `Kafka.spec.cruiseControl.config`
 
-* To change the preset hard goals, specify the desired goals in the `hard.goals` property, using their fully-qualified domain names.
+* To change which hard goals are enforced to execute by Cruise Control, specify the desired goals in the `hard.goals` property, using their fully-qualified domain names.
+
+* To prevent a hard goal from being executed, make sure the goal is not included in the `default.goals` and `hard.goals` list configurations.
```

Comment:
```suggestion
* To prevent execution of a specific hard goal, ensure that the goal is not included in both the `default.goals` and `hard.goals` list configurations.
```